internal/configs/version2/template_helper.go

```diff
@@ -20,17 +20,12 @@ func hasCIKey(key string, d map[string]string) bool {
 	return ok
 }
 
-// toLower takes a string and make it lowercase.
-//
-// Example:
-//
-//	{{ if .SameSite}} samesite={{.SameSite | toLower }}{{ end }}
-func toLower(s string) string {
-	return strings.ToLower(s)
-}
```

Comment:
I removed this function because all it does it call `strings.ToLower` with the exact same parameters, which is unnecessary.